compose-tests/build.gradle.kts

```diff
@@ -30,6 +30,11 @@ android {
     kotlinCompilerVersion = Versions.KotlinCompiler
     kotlinCompilerExtensionVersion = Versions.Compose
   }
+
+  packagingOptions {
```

Comment:
Could you add a comment about why this is needed?

Reply:
It's the same reason it's always needed for most kotlin libraries, I can link to more info but it's pretty standard.